src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        Storage.storage.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());
+    }
```

Comment:
The `BalanceOperation` class should not start with an empty line according to the checklist. Please remove the empty line at the beginning of the class implementation.